### PR TITLE
flake8 fitting

### PR DESCRIPTION
**Pull request by arm61**
#### Description of Work

Fixes flake8 compatibility for the fitting module

#### Testing Instructions

1.
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

File: fitbenchmarking/fitting/plotting/plot_helper.py [hunk 1]
```diff
@@ -8,10 +8,10 @@
 import os
 import numpy as np
 import matplotlib
-matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 import copy
 from fitbenchmarking.utils.logging_setup import logger
+matplotlib.use('Agg')
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/fitting/plotting/plot_helper.py]: This change seems to be breaking.
(This)[https://matplotlib.org/faq/usage_faq.html#what-is-a-backend] suggests that it should be reverted

File: fitbenchmarking/fitting/misc.py [hunk 2]
```diff
@@ -46,8 +46,10 @@ def create_result_entry(problem, status, chi_sq, runtime, minimizer,
     """
 
     if 'fitFunction' in ini_function_def:
-        ini_function_def = ini_function_def.replace('fitFunction', problem.equation)
-        fin_function_def = fin_function_def.replace('fitFunction', problem.equation)
+        ini_function_def = ini_function_def.replace(
```
Comment (AndrewLister-STFC) [inline fitbenchmarking/fitting/misc.py]: Feel free to disagree, but I think this is more readable with:
```
fin_function_def = fin_function_def.replace('fitFunction',
                                            problem.equation)
```

Comment (arm61) [reply fitbenchmarking/fitting/misc.py]: Sorry I disagree 😞, plus what I have is closer to the [black](https://github.com/psf/black) formatting that is becoming more popular in Python

Comment (AndrewLister-STFC) [reply fitbenchmarking/fitting/misc.py]: No problem, I'll resolve this

**Pull request merged**